src/components/AccountDashboard/AccountDashboard.tsx

```diff
@@ -33,14 +40,26 @@ function ProjectsList({ projects }: { projects: ProjectCardProject[] }) {
 }
 
 function ContributedList({ address }: { address: string }) {
-  const { data: contributedProjects, isLoading: myProjectsLoading } =
-    useContributedProjectsQuery(address)
+  const [orderBy, setOrderBy] = useState<Participant_OrderBy>(
+    Participant_OrderBy.volume,
+  )
+
+  const { data, loading: contributionsLoading } = useWalletContributionsQuery({
+    client,
+    variables: {
+      wallet: address.toLowerCase(),
+      orderBy,
+      orderDirection: OrderDirection.desc,
+    },
+  })
+
+  const contributions = data?.participants
```

Comment:
we now use a raw Apollo graphql query to get this data, with no parsing or typing required! only dependencies here are the apollo client and generated types—no custom subgraph types or utils.

Reply:
this is wild